### ISSUE TITLE
Connect to Redis over SSL

### ISSUE THREAD

**Issue by ybo** [TablePlus/TablePlus#652]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): 
- Redis 3.2.6 on AWS

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 
- Build 128

3. The steps to reproduce this issue: 
- I tried to connect to my Redis server in TablePlus but I wasn't able to because there is no SSL option. Is it not supported or did I miss it?

Any help is appreciated!
Thanks

**Comment by huyphams** [comment]
I have researched this before. but the documents say that Redis doesn't support SSL:
```
Redis does not support encryption. In order to implement setups where trusted parties can access a Redis instance over the internet or other untrusted networks, an additional layer of protection should be implemented, such as an SSL proxy. We recommend spiped.
```
https://redis.io/topics/security

Meanwhile, I found this, and it looks potential:
https://github.com/redis/redis-rb/pull/496
I will check it out.

**Comment by ybo** [comment]
Thanks for the quick reply!

On AWS, when you create a new Redis, there is an option to enable/disable "Encryption in-transit". I can connect to a Redis instance that has this option disabled, but I cannot connect to another Redis instance that has this option enabled.

Also, Redis Desktop Manager, another GUI app I use for Redis, has an SSL security option. But then I'm not sure why the doc says it doesn't support SSL.

Thanks for looking into this!

**Comment by saswatds** [comment]
Hi @huyphams, Any update on this issue?

**Comment by lucasltv** [comment]
REDIS TLS PLEASE!!!!!!

**Comment by benjoz** [comment]
Any updates ? 😫😫😫

**Comment by huyphams** [comment]
Let me try again

**Comment by huyphams** [comment]
Hi guys, TablePlus now supports ElasticCache (Redis with TLS), please download this build: https://www.dropbox.com/s/0i0ofnhi88vlxz8/TablePlus.dmg?dl=0

**Comment by benjoz** [comment]
@huyphams it works ! connection is done !
Only things is it looks you don't support yet the "cluster" mode  in order to follow -ASK and -MOVED redirections ? Or maybe I forgot something ?

redis-cli -c
 -c                 Enable cluster mode (follow -ASK and -MOVED redirections).
https://helpmanual.io/help/redis-cli/

<img width="693" alt="Screenshot 2019-06-03 at 12 48 07" src="https://user-images.githubusercontent.com/1252868/58796587-f7ac6000-85fd-11e9-82c5-bdab1ee278e0.png">
